pandas/_libs/tslibs/period.pyx

```diff
@@ -1,11 +1,12 @@
 import warnings
 
+cimport numpy as cnp
```

Comment:
OT: can we be consistent / have a rule about how cimport numpy.

Reply:
i dont know if we have a code check, but im pretty sure we're consistent about this